main/src/ca/uwaterloo/flix/api/lsp/LanguageServer.scala

```diff
@@ -351,6 +354,15 @@ class LanguageServer(port: Int) extends WebSocketServer(new InetSocketAddress("l
     ("id" -> requestId) ~ ("status" -> "success") ~ ("result" -> JArray(allCodeLenses.map(_.toJSON)))
   }
 
+
+  /**
+   * Processes a symbol request
+   */
+  private def processSymbol(requestId: String, uri: String, pos: Position): JValue = {
+
+    // TODO implement the function
+    ("id" -> requestId) ~ ("status" -> "success")
```

Comment:
Could you start by returning a hardcoded list?

Reply:
What should be the return type of this function?

`val result = SymbolProvider.getSymbol(uri, pos) `
 `("id" -> requestId) ~ ("status" -> "success") ~ ("result" -> result.toJSON)`
Here `result` is an hardcoded object of `SymbolInformation`. It is not working and showing this :
![Screenshot from 2021-07-22 12-55-52](https://user-images.githubusercontent.com/45541010/126622411-c3f75d57-3b2a-4385-bb53-07aad04a3bcf.png)

Reply:
> Could you start by returning a hardcoded list?

Why should be there a list? At any location, there can be at most one symbol I guess.

Reply:
https://microsoft.github.io/language-server-protocol/specifications/specification-current/#textDocument_documentSymbol